WalletWasabi.Gui/Controls/LucasPasswordBox.cs

```diff
@@ -0,0 +1,47 @@
+using Avalonia;
+using Avalonia.Controls;
+using Avalonia.Controls.Primitives;
+using Avalonia.Data;
+using Avalonia.Styling;
+using ReactiveUI;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace WalletWasabi.Gui.Controls
+{
+	public class LucasPasswordBox : ExtendedTextBox, IStyleable
+	{
+		Type IStyleable.StyleKey => typeof(LucasPasswordBox);
+		private Button _presenter;
```

Comment:
get set

Reply:
Removed the whole field. Not required. It is part of the control so it will be disposed at the end.